deployer/cluster.py

```diff
@@ -112,14 +112,15 @@ def deploy_support(self):
         print_colour("Provisioning support charts...")
 
         support_dir = (Path(__file__).parent.parent).joinpath("helm-charts", "support")
-        subprocess.check_call(["helm", "dep", "up", support_dir])
-
-        support_secrets_file = support_dir.joinpath("enc-support.secret.yaml")
-        # TODO: Update this with statement to handle any number of context managers
-        #       containing decrypted support values files. Not critical right now as
-        #       no individual cluster has specific support secrets, but it's possible
-        #       to support that if we want to in the future.
-        with get_decrypted_file(support_secrets_file) as secret_file:
+        # subprocess.check_call(["helm", "dep", "up", support_dir])
```

Comment:
I believe this also needs to be restored?

Reply:
Nice catch!!

Reply:
```suggestion
        subprocess.check_call(["helm", "dep", "up", support_dir])
```

Reply:
aha! restored, thank you :)